sdk/src/user.ts

```diff
@@ -1817,29 +1974,73 @@ export class User {
 		return netAssetValue.mul(TEN_THOUSAND).div(totalLiabilityValue);
 	}
 
-	public canBeLiquidated(): {
-		canBeLiquidated: boolean;
-		marginRequirement: BN;
-		totalCollateral: BN;
+	public canBeLiquidated(): AccountLiquidatableStatus & {
+		isolatedPositions: Map<number, AccountLiquidatableStatus>;
```

Comment:
why not just include this on `AccountLiquidatableStatus`? makes it more backward compatible?

Reply:
I guess could do that although I think it would make defining the object for each isolatedPosition more manual. I can try that out though.